app/components/scbd-angularjs-controls/form-control-directives/km-link.js

```diff
@@ -256,7 +265,13 @@ app.directive('kmLink', ['IStorage', function (storage)
 						$scope.editor.urlMissing = true;
 					}
 					if(!$scope.editor.language){
-						$scope.editor.isLanguageSelect = true;
+						var extension = $scope.editor.name.split(".")[1];
+						if (extension != undefined && isImage(extension) ){
```

Comment:
use regex as split  by . may not give good result if the file name is `xyz.avc.jpg` but it can be also done based on mime type, see the previous suggesion
```suggestion
						if (languageOptional() ){
```